docs/en/02_Developer_Guides/00_Model/06_SearchFilters.md

```diff
@@ -36,6 +36,10 @@ These suffixes can also take modifiers themselves. The modifiers currently suppo
 comparison uses the database's default. For MySQL and MSSQL, this is case-insensitive. For PostgreSQL, this is 
 case-sensitive.
 
+Note that all search filters (e.g. `:PartialMatch`) must be written in title case,
```

Comment:
I'm with Matt on this one - the upper/lower casing sensitivity is confusing. Looks like you're introducing a new feature which requires this? Before you had to write `MyField:ExactMatch:case`, now you can write `MyField:case`?